Sources/String.swift

```diff
@@ -81,7 +81,7 @@ internal extension String {
     }
     
     subscript(range: Range<Int>) -> String {
-        return self[characters.index(startIndex, offsetBy: range.lowerBound)..<characters.index(startIndex, offsetBy: range.upperBound)]
+      return String(self[characters.index(startIndex, offsetBy: range.lowerBound)..<characters.index(startIndex, offsetBy: range.upperBound)])
```

Comment:
Honestly? I don't care in the least what form the library takes as long as it works. Those decisions are entirely in your court.